src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/reported/ReportedAdjudicationController.kt

```diff
@@ -52,16 +52,18 @@ class ReportedAdjudicationController(
 
   @Deprecated("to remove on completion of NN-5319")
   @GetMapping(value = ["/{chargeNumber}"])
+  @PreAuthorize("hasRole('VIEW_ADJUDICATIONS')")
   fun getReportedAdjudicationDetails(@PathVariable(name = "chargeNumber") chargeNumber: String): ReportedAdjudicationResponse =
     reportedAdjudicationService.getReportedAdjudicationDetails(chargeNumber).toResponse()
 
   @GetMapping(value = ["/{chargeNumber}/v2"])
+  @PreAuthorize("hasRole('VIEW_ADJUDICATIONS')")
   fun getReportedAdjudicationDetailsV2(@PathVariable(name = "chargeNumber") chargeNumber: String): ReportedAdjudicationResponseV2 =
     reportedAdjudicationService.getReportedAdjudicationDetailsV2(chargeNumber).toResponseV2()
 
   @PutMapping(value = ["/{chargeNumber}/status"])
   @Operation(summary = "Set the status for the reported adjudication.")
-  @PreAuthorize("hasAuthority('SCOPE_write')")
+  @PreAuthorize("hasRole('VIEW_ADJUDICATIONS') and hasAuthority('SCOPE_write')")
```

Comment:
this is an ALO role.  only ALO can set status.  its a bad name for the endpoint, its the review endpoint

Reply:
Good spot! updated with https://github.com/ministryofjustice/hmpps-manage-adjudications-api/pull/467/commits/c2bfe5c16690b7beae059cef905d840845f93795